src/domains/citizen-auth-app/04_function_lollipop.tf

```diff
@@ -44,26 +45,88 @@ locals {
       FIRST_LC_ASSERTION_CLIENT_BASE_URL         = "https://api.io.pagopa.it"
       FIRST_LC_ASSERTION_CLIENT_SUBSCRIPTION_KEY = data.azurerm_key_vault_secret.first_lollipop_consumer_subscription_key.value
     }
+
+    # Scaling strategy
```

Comment:
I suppose this isarethe high load settings, check it out @Krusty93

Reply:
I just moved the ones altready used in fn-lollipop into a shared local.

Reply:
@gunzip yup, very similar, I can't find any difference with a quick look. I think they could start over with the new module by today